opensuse/templates/csv/file_dir_permissions.csv

```diff
@@ -0,0 +1 @@
+/etc,cron.allow,,,0644,cron_allow
```

Comment:
If I understand it well, in OpenSUSE benchmark, there used to be a check for `root` owner and `root` group, but you removed that and replaced that by checking `0644` mode? Why?